libnmstate/nm/nmclient.py

```diff
@@ -44,6 +44,9 @@
 _nmclient = None
 
 
+NM_MANAGER_ERROR_DOMAIN = GLib.quark_to_string(NM.ManagerError.quark())
```

Comment:
My intention in my previous comment was to not execute this at root, which is always a bit nasty. I hoped it can be done in the mainloop object and then use it as a property.

I am not sure if this is passing in the minimalistic unit test image, I will check.

Reply:
It passed the unit tests on the minimalistic image, which I find strange.

The only problem left then is to properly give an error when libnm introspection or other missing parts exist: https://github.com/nmstate/nmstate/blob/master/libnmstate/nm/nmclient.py#L55